core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-list/dot-experiments-list.component.ts

```diff
@@ -60,6 +60,11 @@ export class DotExperimentsListComponent {
     );
     statusOptionList = ExperimentsStatusList;
     confirmDialogKey = CONFIGURATION_CONFIRM_DIALOG_KEY;
+    private dotMessageService: DotMessageService = inject(DotMessageService);
```

Comment:
Do we need this in this case? You have the constructor method declared

Reply:
Removed.